src/pluto/pluto.py

```diff
@@ -278,10 +278,14 @@ def generate_graph(v1v2, v2v3, v1v2_total, v2v3_total, group):
                 fig = generate_graph(
                     v1v2, v2v3, v1v2_total, v2v3_total, group["columns"]
                 )
-                fig.update_layout(title=group["title"], template="plotly_white")
+                fig.update_layout(
+                    title={"text": group["title"], "font": dict(size=24)},
+                    template="plotly_white",
+                )
                 st.plotly_chart(fig)
                 st.info(group["description"])
 
+            st.subheader("Records with Differences by Field")
```

Comment:
I don't think this is the best header here as the table doesn't list records. I think we should replace "Records with" with "Summary of"